test/common/grpc/context_impl_test.cc

```diff
@@ -73,8 +74,8 @@ TEST(GrpcContextTest, ResolveServiceAndMethod) {
   absl::optional<Context::RequestStatNames> request_names =
       context.resolveDynamicServiceAndMethod(path);
   EXPECT_TRUE(request_names);
-  EXPECT_EQ("service_name", absl::get<Stats::DynamicName>(request_names->service_));
-  EXPECT_EQ("method_name", absl::get<Stats::DynamicName>(request_names->method_));
+  EXPECT_EQ("service_name", absl::get<Stats::DynamicSavedName>(request_names->service_));
+  EXPECT_EQ("method_name", absl::get<Stats::DynamicSavedName>(request_names->method_));
```

Comment:
This test case shouldn't fail with `Stats::DynamicName`. Does it?

Reply:
**service_**/**method_** is _absl::variant_ now storing  `DynamicSavedName`, if we try extracting `DynamicName` it's gonna produce `C++ exception with description "Bad variant access"`